linux/devdoc/completion_port_linux_requirements.md

```diff
@@ -66,19 +66,21 @@ MOCKABLE_FUNCTION(, void, completion_port_dec_ref, COMPLETION_PORT_HANDLE, compl
 
 `completion_port_dec_ref` handles the decrement of the reference count and freeing the memory if necessary.
 
-`completion_port_dec_ref` shall decrement the reference count for `completion_port`.
+**COMPLETION_PORT_LINUX_11_008: [** If `completion_port` is `NULL`, `completion_port_dec_ref` shall return. **]**
 
-If the reference count reaches 0, `completion_port_dec_ref` shall do the following:
+**COMPLETION_PORT_LINUX_11_009: [** `completion_port_dec_ref` shall decrement the reference count for `completion_port`. **]**
 
-- wait for the ongoing call count to reach zero.
+**COMPLETION_PORT_LINUX_11_010: [** If the reference count reaches 0, `completion_port_dec_ref` shall do the following: **]**
 
-- increment the flag signaling that the threads can complete.
+**COMPLETION_PORT_LINUX_11_011: [** - wait for the ongoing call count to reach zero. **]**
```

Comment:
>-  [](http://example.com/codeflow?start=36&length=2)

nit: can we make the markdown look nice and reformat all of these as:

` - **SRS[** text **]**`

Instead of having the "-" inside

If you use the SRS extension ALT-F9 over multiple lines (instead of ALT-F8 one at a time) it will preserve those markdown lists